src/components/DatePickerInline/Month.tsx

```diff
@@ -106,7 +106,7 @@ function Month(props: MonthSingleProps | MonthRangeProps | MonthMultiProps) {
     }, [index, isHorizontal, monthStyles, scrollMode]);
 
     const onPressDropdown = useCallback(
-        () => (isHorizontal ? onPressYear(year) : undefined),
+        () => (isHorizontal && onPressYear ? onPressYear(year) : undefined),
```

Comment:
in what case will we not have onPressYear?

Reply:
this is already updated in the new PR @shashank1010